internal/text_distance/text_distance.go

```diff
@@ -0,0 +1,107 @@
+package text_distance
```

Comment:
no underscore in package names please, this would be `textdistance`, but since what the exposed function is `FindSimilarNames`, I would name this package `similartext` and rename `FindSimilarNames` to just `Find` so that it can be used like `similartext.Find(bla, bla)`

Reply:
Both things done (plus `FindFromMap`).